### PR TITLE
Update to 4.5.0 builds of corefx packages

### LINKED ISSUE #661: Update to 4.5.0 builds of corefx packages
Follow up to https://github.com/aspnet/Coherence-Signed/issues/657. We should additionally update repos to reference 4.5.0 version of CoreFx packages.

### PR DESCRIPTION
**Pull request by pranavkm**
Fixes #661

### CODE REVIEW

File: build/dependencies.props [hunk 1]
```diff
@@ -332,56 +331,56 @@
...
+    <ExternalDependency Include="System.Composition.TypedParts" Version="1.2.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="System.Composition" Version="1.2.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="System.Configuration.ConfigurationManager" Version="4.5.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
     <ExternalDependency Include="System.Data.SqlClient" Version="4.3.1" Source="$(DefaultNuGetFeed)" Private="true">
       <!-- This version is used by EF's 1.1 benchmark project -->
       <NoWarn>KRB2004</NoWarn>
       <VariableName>Benchmarks_11_SystemDataSqlClientPackageVersion</VariableName>
     </ExternalDependency>
-    <ExternalDependency Include="System.Data.SqlClient" Version="4.4.0" Source="$(DefaultNuGetFeed)">
+    <ExternalDependency Include="System.Data.SqlClient" Version="4.5.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true">
```
Comment (smitpatel) [inline build/dependencies.props]: Nooo. 😞

File: build/dependencies.props [hunk 2]
```diff
@@ -332,56 +331,56 @@
...
+    <ExternalDependency Include="System.Security.Principal.Windows" Version="4.5.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="System.ServiceProcess.ServiceController" Version="4.5.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="System.Text.Encoding.CodePages" Version="4.5.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="System.Text.Encodings.Web" Version="4.5.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="System.Threading.AccessControl" Version="4.5.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="System.Threading.Tasks.Dataflow" Version="4.9.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="System.Threading.Tasks.Extensions" Version="4.5.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
+    <ExternalDependency Include="System.ValueTuple" Version="4.5.0-preview1-25922-03" Source="$(DotNetCoreFeed)" Mirror="true" />
```
Comment (natemcmaster) [inline build/dependencies.props]: Can reorder the lines to keep the grouping by package feed?

File: build/dependencies.props (hunk 1 above)
Comment (natemcmaster) [reply build/dependencies.props]: @smitpatel is there a reason we shouldn't upgrade?

Comment (smitpatel) [reply build/dependencies.props]: No reason. I got confused by diff and thought that we are upgrading the version which is being used in EF1.1 benchmarks.
All good.

File: build/dependencies.props (hunk 2 above)
Comment (pranavkm) [reply build/dependencies.props]: This seems like a bunch of work for a file that's meant to be auto managed. Would we change it back once we ship 2.1.0?

Comment (natemcmaster) [reply build/dependencies.props]: The file isn't meant to be automanaged, at least not in its current form.

Comment (pranavkm) [reply build/dependencies.props]: Are you manually updating versions of external dependencies? Again, just sounds like a bunch of unnecessary work.

Comment (natemcmaster) [reply build/dependencies.props]: Yes. Hence #577

Comment (pranavkm) [reply build/dependencies.props]: Cool. More of a reason to stop manually curating the order.

**Pull request merged**